projects/packages/search/src/dashboard/components/record-meter/style.scss

```diff
@@ -34,3 +34,17 @@ ul.jp-search-chart-legend {
   text-overflow: ellipsis;
   display:inherit;
 }
+
+.jp-search-notice-box {
+  border: solid .5px lightgrey;
+  margin-left: 5px;
+  padding: 0 5px;
+
+}
+
+.jp-search-notice-box__red {
```

Comment:
It's best to name classes after their usage, rather than including a colour in the class name. A good alternative here might be `.jp-search-notice-box__important`.

https://softwareengineering.stackexchange.com/questions/157219/using-context-based-css-class-names